### DISCUSSION TITLE
Certificato DuckDNS

### DISCUSSION THREAD

**Discussion by Osky9** [indomus/forum#240] [HUB: Home Assistant]
Ciao a tutti,
da stamattina non riesco più ad accedere ad HA dall'esterno (nessun problema per mesi). Da quello che posso capire dal log dell'hadd-on è scaduto il certificato ma non so come posso aggiornarlo (o capire perchè non si aggiorna). Vi copia il log qui di seguito:

```

[18:58:28] WARNING: KO
[18:58:28] INFO: Renew certificate for domains: xxxx.duckdns.org:8123 and aliases: 
# INFO: Using main config file /data/workdir/config
Processing xxxx.duckdns.org:8123
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
  + ERROR: An error occurred while sending post-request to https://acme-v02.api.letsencrypt.org/acme/new-order (Status 400)
Details:
HTTP/2 400 
server: nginx
date: Thu, 29 Apr 2021 16:58:36 GMT
content-type: application/problem+json
content-length: 211
boulder-requester: 111043242
cache-control: public, max-age=0, no-cache
link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
replay-nonce: 0104LIeO-p2c_bqRFUBbZyCF2TXsz1_pGzXsUgr7KoJGw6g

{
  "type": "urn:ietf:params:acme:error:rejectedIdentifier",
  "detail": "Error creating new order :: Cannot issue for \"xxxx.duckdns.org:8123\": Domain name contains an invalid character",
  "status": 400
}
[19:03:39] WARNING: KO

```

Come posso rimediare?
Grazie a tutti

**Comment by snakuzzo** [comment]
C'è scritto il motivo per cui non viene rinnovato il dominio. Ci sono caratteri non validi. Io non uso HassOS...immagino invece che tu lo stia usando. Controlla la configurazione dell'addon

**Reply by Osky9** [reply]
Grazie per la risposta. Si avevo letto anche io questa cosa ma io non ho cambiato nessun carattere e nel mio dominio non ci sono caratteri speciali o cose simili.

**Comment by Osky9** [comment]
Allora cercando un po' in giro ho visto che alcuni utenti hanno avuto il mio stesso problema ma per esempio avevano un "*" nel dominio e quindi gli rimandava l'errore dei caratteri non validi. Ma la mia differenza è che il mio dominio non presenta nessun carattere speciale (solo lettere) e fino ad oggi non ha mai dato problemi. Ho verificato la configurazione dell'addon ma non vedo nulla di strano. A sto punto sarei tentato di cambiare dominio con uno nuovo su duckdns. Secondo voi ha senso?

**Comment by Osky9** [comment]
Alla fine ho risolto togliendo :8123 dalla configurazione dell'addon. L'unica cosa che non capisco è perché prima funzionava anche con la porta specificata.

**Reply by snakuzzo** [reply]
Magari è cambiato qualcosa lato addon.
Leggete sempre i changelog dei progetti
